### PR TITLE
Add Lists getters

### PR DESCRIPTION
**Pull request by apocheau**
I added the ability to retreive informations about a specific list or list all lists.
Just pay attention to the "ListInfo.modules" array, which is not implemented.

### CODE REVIEW

File: src/main/java/com/ecwid/maleorang/method/v3_0/lists/GetListMethod.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.ecwid.maleorang.method.v3_0.lists
+
+
+import com.ecwid.maleorang.MailchimpMethod
+import com.ecwid.maleorang.annotation.*
+
+/**
+ * [Get information about a specific list]
```
Comment (basiliscus) [inline src/main/java/com/ecwid/maleorang/method/v3_0/lists/GetListMethod.kt]: There must be no space or newline here, as it breaks the link in Dokka

File: src/main/java/com/ecwid/maleorang/method/v3_0/lists/CampaignDefaultsInfo.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import com.ecwid.maleorang.annotation.Field
+
+/**
+ * Created by apocheau on 26/08/16.
+ */
+class CampaignDefaultsInfo : MailchimpObject() {
+
+    @JvmField
+    @Field
+    var from_name: String? = null
```
Comment (basiliscus) [inline src/main/java/com/ecwid/maleorang/method/v3_0/lists/CampaignDefaultsInfo.kt]: I'd prefer fields be separated by empty lines

File: src/main/java/com/ecwid/maleorang/method/v3_0/lists/ListInfo.kt [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    @JvmField
+    @Field
+    var beamer_address: String? = null
+    @JvmField
+    @Field
+    var visibility: String? = null
+    /* TODO modules */
+    @JvmField
+    @Field
+    var modules: Array<String>? = null
```
Comment (basiliscus) [inline src/main/java/com/ecwid/maleorang/method/v3_0/lists/ListInfo.kt]: Are you sure that modules is a list of strings? If not, let's remove this field for now. If yes, let's change its type from Array to List

**Comment by basiliscus** [comment]
Hi apocheau,

Thank you for the contribution. I've left some comments and will merge the changes as soon as the issues fixed.

**Comment by apocheau** [comment]
Hello basiliscus and thanks for your reply.
It's the first time i use Kotlin so i appreciated your observations and will follow your guidelines.
For modules, i'll try to find more info about it, otherwise, i'll remove it.
It seems there is a lack of information about this field:

http://developer.mailchimp.com/documentation/mailchimp/reference/lists/#read-get_lists_list_id
"modules": {
			"type": "array",
			"title": "Modules",
			"description": "Any list-specific modules installed for this list.",
			"readonly": true,
			"example": [
				""
			],
			"links": null
		}

Finally, do you have kind of roadmap or stuff you are working on ?

**Comment by basiliscus** [comment]
> Finally, do you have kind of roadmap or stuff you are working on ?

No, I do not have any plans on improving the library. However, contributions are welcome: I will be merging pull requests and releasing new versions.

**Pull request merged**